server/schemas/property.schema.js

```diff
@@ -7,11 +7,18 @@ const mapLocation = Joi.object().keys({
   latitude: Joi.string().label('Map location latitude').optional(),
 });
 
-const product = {
+const address = Joi.object().keys({
+  street1: Joi.string().label('Property Street 1').optional(),
+  street2: Joi.string().label('Property Street 2').optional(),
```

Comment:
I believe this should allow an empty string. I saw this online -
https://stackoverflow.com/a/49581465

```suggestion
  street2: Joi.string().label('Street 2').allow(null, ''),
```